bulbs/videos/templates/videos/series-page.html

```diff
@@ -15,26 +15,67 @@
  <p>Watch The Latest Episode</p>
 </section>
 
-<figure class="video-container">
-  <div class="content">
-    <bulbs-video
-      twitter-handle="{{ twitter_handle }}"
-      data-videohub-base="{{ VIDEOHUB_BASE_URL }}"
-    >
-    </bulbs-video>
-  </div>
-</figure>
+<div class="latest-episode-container">
+  <figure class="video-container">
+    <div class="content">
+      <bulbs-video
+        twitter-handle="{{ twitter_handle }}"
+        data-videohub-base="{{ VIDEOHUB_BASE_URL }}"
+      >
+      </bulbs-video>
+    </div>
+  </figure>
 
-<div id="latest-video-title"></div>
+  {% if channel.slug == 'the-av-club' %}
+    <div id="latest-video-title"></div>
+  </div>
+  {% endif %}
+  {% if channel.slug == 'the-onion' %}
```

Comment:
One of these conditionals should be the default and the other should be overridden in the implementing property template with a `{% block %}` extension